hud-data/weirdcorehud.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "WeirdcoreHUD",
+  "author": "Tomma",
+  "contributors": [
+    "Yara"
+  ],
+  "releaseDate": "2024 04 06",
+  "resources": [
+    "https://youtu.be/hlAn1QJPX7o"
+  ],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "crosshairs",
+    "minmode",
+    "materials",
+    "customization"
+  ],
+  "tags": [
+    "vaporwave"
+  ],
+  "repo": "https://github.com/zTomma/WeirdcoreHUD",
+  "hash": "7516de9cbdf71cfc0c07268651f083e7d2a15d88",
+  "prerelease": false,
+  "social": {
+    "steam_profile": "76561199089787762",
+    "twitter": "@zTomma_",
```

Comment:
May work either way but I believe most huds here list their twitter without the `@` (is this something that the checks would pick up @mastercoms?)